keywords.go

```diff
@@ -145,6 +160,41 @@ var (
 		"link",
 		"tar_time",
 	}
+
+	// BsdKeywords is the set of keywords that is only in the upstream FreeBSD mtree
+	BsdKeywords = []string{
+		"cksum",
+		"device",
+		"flags",
+		"ignore",
+		"gid",
+		"gname",
+		"link",
+		"md5",
+		"md5digest",
+		"mode",
+		"nlink",
+		"nochange",
+		"optional",
+		"ripemd160digest",
+		"rmd160",
+		"rmd160digest",
+		"sha1",
```

Comment:
we should probably have the 'shortened' digest keywords to output the full digest name, as upstream mtree uses the full version, such as `sha1digest`, `md5digest`, etc.